uvnet/models.py

```diff
@@ -148,42 +148,33 @@ def training_step(self, batch, batch_idx):
         inputs.edata["x"] = inputs.edata["x"].permute(0, 2, 1)
         logits = self.model(inputs)
         loss = F.cross_entropy(logits, labels, reduction="mean")
-        self.log("train_loss", loss)
+        self.log("train_loss", loss, on_step=False, on_epoch=True)
```

Comment:
Also made some tweaks to log per epoch using lightning features rather than manually.